tests/integration/test_backups.py

```diff
@@ -355,12 +352,9 @@ async def test_restore_on_new_cluster(ops_test: OpsTest, github_secrets) -> None
 
     # Wait for the restore to complete.
     async with ops_test.fast_forward():
-        await wait_for_idle_on_blocked(
-            ops_test,
-            database_app_name,
-            0,
-            S3_INTEGRATOR_APP_NAME,
-            ANOTHER_CLUSTER_REPOSITORY_ERROR_MESSAGE,
+        unit = ops_test.model.units.get(f"{database_app_name}/0")
+        await ops_test.model.block_until(
+            lambda: unit.workload_status_message == ANOTHER_CLUSTER_REPOSITORY_ERROR_MESSAGE
         )
```

Comment:
Model fails to idle with the new snap. I tried with one minute ffwd interval, but it didn't help.